packages/cli/src/lib/implementation/only-plugins.utils.unit.test.ts

```diff
@@ -86,10 +90,14 @@ describe('filterCategoryByPluginSlug', () => {
         verbose: true,
       },
     );
-    expect(console.info).toHaveBeenCalledWith(
-      expect.stringContaining('"category1" is ignored'),
+    const logs = ui()
+      .logger.getRenderer()
+      .getLogs()
+      .map(({ message }) => message);
+    expect(logs[0]).toEqual(
+      expect.stringContaining('Category "category1" is ignored'),
     );
-    expect(console.info).toHaveBeenCalledWith(
+    expect(logs[0]).toEqual(
       expect.stringContaining('skipped plugin "plugin2"'),
     );
```

Comment:
The logs assertion is very verbose, it would be nice if we could extract it. Could a helper function in `test-utils` (e.g. `getLogsMessages(): string[]`), or even a [custom matcher](https://vitest.dev/guide/extending-matchers) (e.g. `expect(ui()).toHaveLogged('...')` or `expect(ui()).toHaveLoggedNth(1, '...')` or `expect(ui()).not.toHaveLogged()`) :thinking: - cc @Tlacenka

Reply:
I implemented the utils method and created a issue for the custom matcher #552

Reply:
Thanks, that's a well-described issue :slightly_smiling_face: